sass/css/app.scss

```diff
@@ -0,0 +1,3 @@
+@import 'themes/themes.scss';
+@import 'themes/themify.scss';
+@import 'themes/themed.scss';
```

Comment:
Is it possible to diversify these file names a bit? Having two files of 'themes' and 'themed' is somewhat confusing

Reply:
which one of them we are actually using? if we will need to use all 3 then we need better names, but if we are only using one as a gateway for the other, then its fine